<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimpleLensCorrectionEditorComp</class>
 <widget class="QWidget" name="CSimpleLensCorrectionEditorComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>72</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Optical Center</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="CenterXSB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="CenterYSB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Correction</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="3">
    <widget class="QPushButton" name="CalibrateButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="ScaleSB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="CorrectionFactorSB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>px</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CalibrateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
